rfcs/text/0000-conceptual-data-model.md

```diff
@@ -0,0 +1,111 @@
+
+
+### User Stories
+
+As a DCP developer, to develop software to manage versioned data,
+I would like to understand how to interpret the data and communicate with other developers in a clear and precise manner.
+
+As a data operations engineer, to construct a data release, I would like to understand what specific data and metadata instances to include.
+
+As a bioinformatician, to replicate an experiment, I would like to understand the provenance of the relevant data files from a data release.
+
+## Detailed Design
+
+The experimental metadata is structured as DAG of instances, with basic categories of *Project*, *Biomaterial*, *Protocol*, *Process*, and *File*.  Each of these types may have more detailed derived types.  This structure is defined in more detail in [Overview of Metadata Structure](https://github.com/HumanCellAtlas/metadata-schema/blob/master/docs/structure.md).
+
+Conceptually, the full, versioned metadata graph consists of all metadata instance versions in the DCP.  However, there are multiple ways to traverse the graph based on the question being asked:
+
+
+* Head graph: The top-down traversal of the most current version of metadata instances.  This does not include obsolete instances.  This view of the graph is normally used for browser and selection of the latest results
+* Provenance graph: This is the reverse-traversal graph from a specific version of a metadata instance, visiting the exact versions of preceding metadata instances.  It records the context of an experiment when it was run.
+* Amended provenance graph: The provenance graph with corrections that are inconsequential, such as correcting natural language descriptions.  This graph is used to produce data releases.
+
+
+The head graph is the graph of the latest version of all the metadata, excluding those that are obsolete due to a restructuring update or those that are flagged as erroneous.  This is the view normally used in selecting data sets and exploring the atlas.  Traversing from the roots to result in a branching set of paths to the leaves.  Normally, this graph would be generated by traversing via UUID, ignored the version timestamps.
+
+The providence graph gives the exact history of data and metadata as it existed at the time a given process was run.  This view provides the ability to analyze and understand an experiment.  It is a reverse traversal from leaves to roots following FQIDs of the input to each metadata instance as it existed at the time the leaf was created.  Since metadata indirectly upstream of a process may change without intermediary processes being rerun, the providence links are not simply the version of the direct input to a node at the time each metadata entity was created.  The upstream metadata change may have triggered a downstream rerun without intermediate processes being rerun.
```

Comment:
Should this be `Provenance` instead of `providence`?

Reply:
I am really unclear on what this sentence means `the providence links are not simply the version of the direct input to a node at the time each metadata entity was created.  The upstream metadata change may have triggered a downstream rerun without intermediate processes being rerun.`